brianchandotcom/liferay-portal
# LPS-170198 Remove wiki display context interfaces

# What is this about?

https://issues.liferay.com/issues/browse/LPS-170198

The Wiki display context interfaces were not used, and as an API, it was largely unmantained. Make all display contexts private to the web module and remove the interfaces and support classes.

# How do I test it?

This should not change how wiki works. As long as all tests pass, we're good.

@brianchandotcom resending this pull directly to you since after a few retries we confirmed this pull is not related to any of the failing tests in https://github.com/liferay-lima/liferay-portal/pull/3797
